modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/ClusterMetricsImpl.java

```diff
@@ -379,12 +379,12 @@ public ClusterMetricsImpl(GridKernalContext ctx, long nodeStartTime) {
 
     /** {@inheritDoc} */
     @Override public float getBusyTimePercentage() {
-        return 1 - getIdleTimePercentage();
+        return Math.max(100 - getIdleTimePercentage(), 0);
```

Comment:
Let's replace 100 with 1 here to leave public contract unchanged.

Reply:
Again, the method is called " Percentage"